server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileBackuper.java

```diff
@@ -47,6 +47,8 @@
 
 public class QProfileBackuper implements ServerComponent {
 
+  public static final String MEDIA_TYPE_XML = "text/xml";
```

Comment:
Actually, I was expecting to be able to use Guava's MediaType class, but it's only available since Guava 12 :stuck_out_tongue_winking_eye: